json/src/fs2/data/json/internal/package.scala

```diff
@@ -17,10 +17,80 @@ package fs2
 package data
 package json
 
+import cats._
+
 package object internals {
 
   private[internals] val hexa = "0123456789abcdef"
 
-  private[internals] type Result[F[_], In, Out] = Option[(Chunk[In], Int, Stream[F, In], Out)]
+  private[json] type Result[F[_], In, Out] = Option[(Chunk[In], Int, Stream[F, In], Out)]
+
+  private[json] def skipValue[F[_]](chunk: Chunk[Token],
+                                    idx: Int,
+                                    rest: Stream[F, Token],
+                                    depth: Int,
+                                    chunkAcc: List[Token])(
+      implicit F: ApplicativeError[F, Throwable]): Pull[F, Token, Result[F, Token, List[Token]]] =
```

Comment:
I wonder (in general for fs2-data) whether it would make sense to use `RaiseThrowable`…it would allow describing the streams in terms of `Fallible` which makes it a bit more general and detached from the call site. Not sure it's worth it though, what do you think?

Reply:
Maybe, I can investigate, but it definitely belongs to another PR.